packages/lambda-tiler/src/routes/tile.style.json.ts

```diff
@@ -101,6 +101,7 @@ export async function tileSetToStyle(
 export async function styleJsonGet(req: LambdaHttpRequest<StyleGet>): Promise<LambdaHttpResponse> {
   const apiKey = Validate.apiKey(req);
   const styleName = req.params.styleName;
+  const excludeLayers = req.query.getAll('e');
```

Comment:
could we have something slightly more readable than `e` ? 

`layerExclude=` or maybe even `exclude=`?